synapse/models/infotech.py

```diff
@@ -1743,6 +1757,57 @@ def getModelDefs(self):
                 )),
                 ('it:av:signame', {}, ()),
 
+                ('it:av:scan:result', {}, (
+
+                    ('time', ('time', {}), {
+                        'doc': 'The time the scan was run.'}),
+
+                    ('verdict', ('int', {'enums': suslevels}), {
+                        'doc': 'The scanner provided verdict for the scan.'}),
+
+                    ('scanner', ('it:prod:softver', {}), {
+                        'doc': 'The scanner software used to produce the result.'}),
+
+                    ('scanner:name', ('it:prod:softname', {}), {
+                        'doc': 'The name of the scanner software.'}),
+
+                    ('signame', ('it:av:signame', {}), {
+                        'doc': 'The name of the signature returned by the scanner.'}),
+
+                    ('target:file', ('file:bytes', {}), {
+                        'doc': 'The file that was scanned to produce the result.'}),
+
+                    ('target:proc', ('it:exec:proc', {}), {
+                        'doc': 'The process that was scanned to produce the result.'}),
+
+                    ('target:host', ('it:host', {}), {
+                        'doc': 'The host where the scan was run.'}),
```

Comment:
is this docstring correct? `target:host` seems slightly out of place as the name for the host where it was run, since the other `target:*` are the thing that was scanned

Reply:
I suppose this is meaning a locally run AV scan, i was thinking of a host execution type `:host` that was the thing running the scans on other targets

Reply:
Yea, i struggled with it a bit. I can tweak the wording to be a bit simpler.